apps/imgviewer.c

```diff
@@ -100,143 +116,174 @@ void resize_finish(int w, int h) {
 	yutani_flip(yctx, window);
 }
 
+int file_exists(char * filename){
```

Comment:
Why wrap `access` like this? It servers no purpose. Also, I was quite clear in my implementation advice in the original ticket that using `access` for this was not the right approach. This introduces a class of bugs called "time of check to time of use" - the file could be deleted between this call and the `fopen` from the graphics library.